terraform/modules/common/instance-profile/main.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_iam_instance_profile" "profile" {
+  name = "${var.role}-${var.product}-${var.environment}-instance-profile"
+  role = "${var.role}"
+}
```

Comment:
i don't see where this module gets used. it also looks far too small to be something worth extracting into a module